certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Board Columns",
+        "operationId": "GetColumns",
+        "parameters": [
+          {
+            "name": "board_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The board to get the columns for.",
+            "x-ms-summary": "Board ID"
+          }
+        ],
+        "description": "Gets all columns for a specified board."
+      }
+    },
+    "/api/v2/boards/{board_id}/lanes": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "lane_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "lane_id"
+                      },
+                      "workflow": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "workflow"
+                      },
+                      "parent_lane_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "parent_lane_id"
+                      },
+                      "position": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "position"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "description"
+                      },
+                      "color": {
+                        "type": "string",
+                        "description": "color"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Lanes",
+        "operationId": "GetLanes",
+        "parameters": [
+          {
+            "name": "board_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The board to get the lanes for.",
+            "x-ms-summary": "Board ID"
+          }
+        ],
+        "description": "Gets all lanes for a specified board."
+      }
+    },
+    "/api/v2/cardTypes": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@dimitar-kanbanize Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.